site/static/linux/centos/tetrate-getenvoy.repo

```diff
@@ -1,15 +1,21 @@
 [tetrate-getenvoy-stable]
 name=Tetrate GetEnvoy - Stable
-baseurl=https://tetrate.bintray.com/getenvoy-rpm/centos/$releasever/$basearch/stable/
+baseurl=https://rpm.dl.getenvoy.io/public/rpm/el/8/$basearch
```

Comment:
are we make all these 3 files identical? If so we should just have one, and point to same file in docs.

Reply:
fair enough. Let me move all of them under `rpm` and see if I can hack the docs.